### PR TITLE
added simple_triplet_matrix() converter

### PR DESCRIPTION
**Pull request by sgsokol**
Address issue #194

### CODE REVIEW

File: inst/unitTests/runit.sparseConversion.R [hunk 1]
```diff
@@ -674,12 +674,19 @@ if (.runThisTest) {
...
+        ## simple_triplet_matrix from package slam
         test.as.stm2dgc <- function() {
-            ## simple_triplet_matrix from package slam
             stm <- as.simple_triplet_matrix(diag(2))
             dgc <- as(as(stm, "matrix"), "dgCMatrix")
             checkEquals(dgc, asSpMat(stm), msg="stm2dgc")
         }
+        test.as.stm2stm <- function() {
+            if ("package:Matrix" %in% search())
+                uppressMessages(unloadNamespace("Matrix"))
```
Comment (eddelbuettel) [inline inst/unitTests/runit.sparseConversion.R]: That is a type:  `suppressMessages()`.

I am not a fan of trying to unload packages. I don't think it works reliably.  What are you trying to do?

Comment (sgsokol) [reply inst/unitTests/runit.sparseConversion.R]: Just to check that even without Matrix loaded, the conversion stm -> arma -> stm works well.
If unloading is a concern, we can do without it but we have to have a faith that it works without Matrix.

Comment (eddelbuettel) [reply inst/unitTests/runit.sparseConversion.R]: That is how the unit tests work (eg on Travis):  if not explicitly loaded, it is not there. I'd remove the line with the typo and the preceding test.

Comment (sgsokol) [reply inst/unitTests/runit.sparseConversion.R]: But it is explicitely loaded in runit.sparseConversion.R:34
```
.runThisTest <- suppressMessages(require(Matrix))
```

Comment (eddelbuettel) [reply inst/unitTests/runit.sparseConversion.R]: That's the point. We _need_ Matrix for the existing conversions.

Comment (sgsokol) [reply inst/unitTests/runit.sparseConversion.R]: Right. That's why I have unloaded it just for one test and reloaded it immediately after. But as I already said, it is OK for me to do without unload/reload. I have checked on my side that it worked without Matrix.

**Pull request closed without merging**